.buildkite/pipeline.yml

```diff
@@ -26,14 +26,29 @@ steps:
         environment:
           - PYTHONDONTWRITEBYTECODE=1
 
+  - label: ":docker: :hadolint:"
+    command: "find -name 'Dockerfile*' -print0 | xargs -0 hadolint"
+    plugins:
+       docker#v3.0.1:
+          image: "hadolint/hadolint:latest-debian"
+
+  - label: ":docker: publish numbered images"
+    command: ".buildkite/steps/manage-images.sh build-publish-numbered"
```

Comment:
What's the motivation for doing this separation? For the other places, it's been motivated by wanting to run tests on the temporary images but that's not currently the case here (it would probably good here as a future enhancement... it'd be nice to check that `stellargraph`/`tensorflow`/etc. all get installed right).

Reply:
Currently it's so the image can be built in parallel with the other steps, so the build is faster—but also in case other tests may be used